src/DrawTogether.UI/Client/Components/Curve.razor

```diff
@@ -11,16 +13,26 @@
     [Parameter]
     public string Stroke { get; set; } = "#ff0000";
 
-    protected static string PathData(List<Point> Points)
+    protected override bool ShouldRender()
+    {
+        if (_points.Count() != Points.Count)
+        {
+            _points = Points.ToArray();
+            return true;
+        }
+        return false;
+    }
+
+    protected static string PathData(Point[] Points)
     {
     // Parameter for smoothness of path in interval [0, 0.5]
         double smoothness = 1.0 / 3.0;
 
         var result = "";
-        if (Points.Count >= 2)
+        if (Points.Count() >= 2)
```

Comment:
Should probably change this to `Points.Length` in order to avoid a LINQ invocation, but that's small potatoes